pages/linux/runsvdir.md

```diff
@@ -0,0 +1,15 @@
+# runsvdir
+
+> Run an entire directory of services.
+
+- Start and manage all services in a directory as yourself:
```

Comment:
"as the current user" might sound better.

Reply:
Fixed